connector.schema.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
```suggestion
    "$schema": "http://json-schema.org/draft-07/schema#",
    "$id": "https://adobe.io/schemas/aio/connectors/0-0-1",
```

Reply:
See **SchemaVer** for the full suggestion on versioning the schema itself: https://snowplowanalytics.com/blog/2014/05/13/introducing-schemaver-for-semantic-versioning-of-schemas